src/update.cpp

```diff
@@ -0,0 +1,186 @@
+#include "update.h"
+
+#include <atomic>
+
+#include "network.h"
+#include "filesystem.h"
+#include "vitaPackage.h"
+#include "zip.h"
+#include "Views/dialogView.h"
+
+// FIXME: tmp url for deving
+#define BASE_ADDRESS "https://raw.githubusercontent.com/robsdedude/vhbb/autoUpdate/"
```

Comment:
Note that this has to be change when you accept the PR.

I've created a branch `updateTestBranch` in me repo that offers a `binary.bin` containing version `01.00` and a vpk that opens a dialog at start telling you the update worked. In case you want to test the mechanism.

Reply:
Seems broken here:
![image](https://user-images.githubusercontent.com/2824100/53716534-8a016400-3e55-11e9-91ad-85bae6d5db9c.png)